examples/lidar.py

```diff
@@ -0,0 +1,53 @@
+import datetime as dt
+import camille
+
+start_date = dt.datetime(2018, 1,1)

```

Comment:
This feels a bit iffy, I guess this should be configurable? No?

Reply:
Could be, but this would likely end up in a jupyter notebook until it’s automated

Reply:
Sure. Maybe still reasonable to make it an obvious customisation point (i.e. a function to set it or whatever), but I guess it's fine. Your call.